_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -0,0 +1,86 @@
+
+## Using a snapshot for search
+
+Before searchable snapshots, searching data within a snapshot required restoring the entire snapshot to the OpenSearch cluster node. This involved expanding your infrastructure, which is costly and time-consuming. 
+
+The searchable snapshots feature caches frequently used data segments in cluster nodes, using the Least Recently Used (LRU) caching strategy to remove the least used data segments from the cluster nodes in order to make space for frequently used ones. The data segments downloaded from snapshots on block storage reside alongside the general indexes of the cluster nodes. The local storage of the node is used for caching the snapshot data. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot. Therefore, you can search snapshots stored in lower-cost storage, like Amazon S3, with acceptable performance. 
+
+The following image shows the creation of an index using a remote storage option, such as Amazon S3.
+
+![Searchable Snapshots](/assets/media/blog-images/2023-04-27-searchable-snapshots/searchable-snapshot.png){: .img-fluid}
+
+## Enabling searchable snapshots in OpenSearch
+
+To enable searchable snapshots, reference the [instructions](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/) in the OpenSearch documentation.
+
+## Performance
+
+Comparing the performance of searchable snapshot queries and local index queries on the `stackoverflow` dataset, the initial run of searchable snapshots took approximately twice as long as the local index. However, subsequent queries showed similar performance for both searchable snapshots and local index because searchable snapshots used cached data. 
+
+The following table presents the time in seconds taken to scan approximately 36 million documents in the [`stackoverflow`](https://www.kaggle.com/datasets/stackoverflow/stackoverflow) dataset using the Scroll API in a single-threaded test runner.
+
+| | **Local index** | **Searchable snapshot** |
+|:--- | --- | --- |
+| First scan | 588 | 1233 |
+| Repeat scan | 590 | 641 |
+
+Note that searchable snapshots cannot be deleted until the `searchable` flag on these snapshots is removed. Follow the snapshot [configuration steps](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/) in order to remove the flag.
```

Comment:
I may be missing it somewhere but was unable to find the `searchable` flag being referred here in the linked doc or the searchable snapshot page. Would be better to clarify this

Reply:
nit: ` all searchable snapshots must be created` --> ` all searchable snapshots created`